src/LandCover.js

```diff
@@ -31,6 +32,7 @@ class LandCoverServer {
       geometry: position,
       geometryType: 'esriGeometryPoint',
       inSR: 4326,
+      outSR: this.spatialReference,
```

Comment:
Wisconsin's service defaulted to a different spatial reference. This ensures that it matches the referenced used in `project()`.